### PR TITLE
Fifty

### PR DESCRIPTION
**Pull request by J-Wass**
Added 50/50 detection as an "intensive event stat" since it takes some computing ability. Works pretty well and can lead to dunk detection in the near future.

### CODE REVIEW

File: api/stats/events.proto [hunk 1]
```diff
@@ -46,6 +47,11 @@ message Hit {
 
     optional bool clear = 23;
 
+    // Scales from 100 to 0;
+    // 100 indicates an opponent was right on top of this hit.
+    // 0 indicates the hit was made with little to no defending pressure.
+    optional int32 pressure = 24;
```
Comment (dtracers) [inline api/stats/events.proto]: Should this exist in 50s?

Comment (J-Wass) [reply api/stats/events.proto]: Ah, this is supposed to have a diff base with the hit-pressure branch. Not sure how to indicate that on git. Can fix.

**Comment by Sciguymjm** [comment]
Keep in mind @maintainers this PR includes #217 in its commits so don't merge this one until #217  is merged in.

**Comment by dtracers** [comment]
This has some conflicts that needs resolving it looks like

**Pull request merged**